client/ide/lib/index.coffee

```diff
@@ -258,6 +258,14 @@ class IDEAppController extends AppController
 
   setActiveTabView: (tabView) ->
 
+    activePane = tabView.getActivePane()
+
+    for view in @ideViews
+      for tab in view.holderView.tabs.subViews
+        if tab is activePane?.tabHandle
+        then activePane.tabHandle.setClass 'focus'
+        else tab.unsetClass 'focus'
```

Comment:
we shouldn't need something like that, currently there is a style change on focus in somewhere else. I might be wrong but this looks weird to me.